h5py/tests/test_dataset.py

```diff
@@ -816,14 +816,18 @@ def test_no_chunks(self):
     def test_track_times(self):
         orig = self.f.create_dataset('honda', data=np.arange(12),
                                      track_times=True)
+        self.assertEqual(orig.track_times, True)
```

Comment:
It would be nicer to use `assertTrue`/`assertFalse` or `assert` and `assert not`

Reply:
ok, will do.